_posts/2019-09-30-team-working-agreements.md

```diff
@@ -0,0 +1,261 @@
+The brainstorm for a team working agreement can fill this void.
+
+During the team agreement brainstorm the team looks at _how_ they work, instead of looking at _what_ work they're
+doing. It's similar to what you might get out of a retro but at a higher level. It offers time to discuss "soft"
+skills that you don't often discuss, in a context that you don't usually discuss them - with other individual
+contributors, and with your closest collaborators. (ADAM: can this paragraph be reorganized to contrast the
+proactive nature of TWAs to the retroactive nature of retros?)
+
+<aside class="dialogue">
+  <div class="answer">
+    <img src="/images/dialogue/steve.jpg">
+    <p class="intro">Steve Hicks says...</p>
+    <p>
+      When joining a new team, there's often a period of "feeling it out" - where I'm afraid to verbalize an
+      opinion because I'm not sure how the team feels about it. The team working agreement process has definitely
+      helped lessen this feeling.
+    </p>
+    <p>
+      There's a [great talk by Katherine Wu named "Ask vs Guess Culture"](https://www.youtube.com/watch?v=FxTXf6qyaGk). 
+      "Guess" culture is when you try to guess what a person wants or is thinking; "Ask" culture is when you flat out 
+      ask them. Team agreement brainstorms are a really neat way to *ask* your team how they want to work, instead of
+      spending many sprints trying to guess.
+    </p>
+    <p>
+      So far, I have seen the most value out of *the actual brainstorm* than the working agreement itself. My teams 
+      have learned so much about each others' habits and values.
+    </p>
+  </div>
+  <div class="answer">
+    <img src="/images/dialogue/adam.jpg">
+    <p class="intro">Adam Iskounen says...</p>
+    <p>
+      TODO Adam: put this in your own words/expand/do what you want here
+      The process has helped confirm that the team shared interest in team dynamics & the process of collaboration 
+      across the team. As a new member it's nice to know that information.
+    </p>
+  </div>
+</aside>
+(ADAM: discuss the value/impact of the brainstorm here. perhaps this is where I can discuss how I didn't think our retros were effective during onboarding rotations)
+
+### It can lead to better engagement in sprint activities.
+
+One of our teams established in their working agreement that sprint meetings were valuable. This might seem too
+obvious to put in a team working agreement, but it portrays clearly to everyone that they should attend all sprint
+meetings and be actively engaged.
+
+The same team got a lot out of making slogans for the items in their working agreement. Phrases like "Pair by
+default" and "Incremental improvement over consistency" were introduced during the brainstorm. They've since become
+mantras for the team to refer to during sprint work and ceremonies. Team values and habits solidify through this
+kind of repetition.
```

Comment:
I'd love to see a compendium of these slogans, so that future teams can select from that menu of practices.